source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/IActionLayoutRenderer.java

```diff
@@ -10,14 +10,15 @@
 import io.adaptivecards.objectmodel.BaseActionElementVector;
 import io.adaptivecards.objectmodel.HostConfig;
 import io.adaptivecards.renderer.actionhandler.ICardActionHandler;
+import io.adaptivecards.renderer.typeaheadsearch.IChoicesResolver;
```

Comment:
I think this import may not be needed anymore

Reply:
yes, removed